pydax/_high_level.py

```diff
@@ -185,9 +185,11 @@ def load_dataset(name: str, *,
     2 2010-01-01 03:00:00               5.0                33.0
     """
 
-    schema = export_schemata().schemata['datasets'].export_schema('datasets', name, version)
+    schemata = export_schemata().schemata['datasets']
+    schema = schemata.export_schema('datasets', name, version)
+    schema_name = schemata.export_schema().get('name', 'default')
```

Comment:
I changed it to `schemata_name`. For some reason I used it in the test but not here. One thing we can do is to bring some uniformity of the naming, i.e., using `schemas` as a generic plural form of `schema` and `schemata` as an aggregation of schemas in a single schema file (or set). What do you think?

Reply:
Schema vs schemata has been my biggest headache on this project haha. 

I think up to this point we've been doing the opposite of what you're suggesting since we've been referring to our schema files as schemata (e.g. `_schemata`, `SchemaManager.schemata`, `export_schemata`). So should we instead continue to do that and refer to 

> an aggregation of schemas in a single schema file

as `schemas` or `datasets_schema`?

Reply:
I agree we are in a mess here. I changed this to `dataset_schema_name` to be consistent with  https://github.com/CODAIT/pydax/blob/56c59393ba5d9dda7e01a144bb29b6f48dea1aca/pydax/_config.py#L35 so that we can locate these spots easily later on.

How about we make a separate PR to address this? We better come up with a better name for the content of a schema file. Maybe a "schemagg"? (Schemata Aggregated)

Reply:
I think the major scenarios to consider can be summed up using `export_schemata`:

1. `export_schemata().schemata` 
    * Full datasets, licenses, formats schemas
    * Currently referred to as `schemata`
    * I propose we keep the way it is

2. `export_schemata().schemata['datasets']`
    * Full datasets schema
    * Currently referred to as `schemata` (in your PR)
    * I propose we rename to `datasets_schemata`, `licenses_schemata`, `formats_schemata` as applicable

3. `export_schemata().schemata['datasets'].export_schema('datasets')`
    * Aggregated dataset schemas
    * Currently referred to as `dataset_schema`
    * I propose we rename to `dataset_schemas`, `license_schemas`, `formats_schemas` as applicable

4. `export_schemata().schemata['datasets'].export_schema('datasets', name, version)`
    * Single dataset schema
    * Currently referred to as, when generic `dataset_schema`, or specific as e.g. `gmb_schema`
    * I propose we keep the way it is

Any other cases I'm missing? Case 2. and 3. I think is what is tripping us up. Open to suggestions for those. Might want to also change `Config().DATASET_SCHEMA_URL` to `Config().DATASET_SCHEMA_FILE_URL` to be more explicit. There's also scenarios like `SchemaDict` and `subdataset_schema` to consider.

Reply:
What do you mean by renaming? For 2, do you mean adding properties named `datasets_schemata`, `licenses_schemata`, `formats_schemata` to `SchemaManager`? Similar for 3 (e.g., `export_schemata().datasets_schemata.dataset_schema`).

I agree with the change in `Config`. I'm also good with 1 and 4.

Reply:
Would you merge this PR? This seems like a different issue (which we can try to resolve immediately once we have a solution)

Reply:
sure thing